packages/inter-protocol/src/reserve/params.js

```diff
@@ -2,8 +2,10 @@
 
 import { CONTRACT_ELECTORATE, ParamTypes } from '@agoric/governance';
 
-const makeReserveTerms = (poserInvitationAmount, timer) => ({
-  timer,
```

Comment:
Motivation is the caller never provided the argument and it's now detected. The runtime impact is that property will be omitted from the return instead of being undefined. I don't see anything checking for the key presence specifically.

Reply:
Given lack of value at the caller site, that change sounds reasonable